app/presenters/cost_summary/disbursements.rb

```diff
@@ -19,7 +19,7 @@ def rows
     end
 
     def total_cost
-      @total_cost ||= disbursement_forms.sum(&:total_cost)
+      @total_cost ||= disbursement_forms.filter_map(&:total_cost).sum
```

Comment:
pretty much.. this is a guard for bad data - it should really be possible with this change to the flow... but there are a couple of edges cases (related to using the back button) that could potentially get the user into a bad state. 

Hence I have added this to avoid errors on the page if these conditions are met (most likely will only occur during testing scenarios)

Reply:
the below is a more long hand form of what it's doing
```
disbursement_forms.map(&:total_cost).compact.sum
```